lib/web/jquery/jquery.cookie.js

```diff
@@ -47,7 +47,7 @@
                 options.path    ? '; path=' + options.path : '',
                 options.domain  ? '; domain=' + options.domain : '',
                 options.secure  ? '; secure' : '',
-                options.samesite  ? '; samesite=' + options.samesite : 'lax',
+                '; samesite=' + (options.samesite ? options.samesite : 'lax'),
```

Comment:
Hello @ihor-sviziev.
Great changes.
But I have a question.
I am not sure we can change the third-party libraries.
A couple years ago we shouldn't do it.

Reply:
Actually they were changed in scope of https://github.com/magento/magento2/commit/50e7b94e4fc181ae42150ae52b30e608cba7e426#diff-d0542fed99e2456d704d0d4b4fed14c9c2dedc98b9fb02a3fefc1c229c4f9e14

Reply:
Ah, ok.
Thank you for your answer @ihor-sviziev.
Waiting for finishing tests.